chevah_build

```diff
@@ -443,7 +443,7 @@ case $OS in
         EXTRA_LIBRARIES=""
         # A patched PyCrypto is packed and uploaded to our PyPI server.
         PIP_LIBRARIES="$PIP_LIBRARIES \
-            pycrypto==${PYCRYPTO_VERSION}chevah1 \
```

Comment:
Sorry, but why would you not want our patched PyCrypto? It has quite a few security patches…

Reply:
This was just to align with the other libraries.
As on Linux we were not using the patched version.

This needs to be fixed via the `${PYCRYPTO_VERSION}` which should include our patched version.

https://github.com/chevah/python-package/pull/144/files#diff-02f039358640626a68241789f3e4fc2dL32

So instead of using 

```sh
PYCRYPTO_VERSION=1.2.3
pip install pycrypto==${PYCRYPTO_VERSION}chevah1
```

we should have something like


```sh
PYCRYPTO_VERSION=1.2.3.chevah1
pip install pycrypto==${PYCRYPTO_VERSION}
```

Does it make sense?

Reply:
We are using the patched version on Linux and other OS'es as well. It's just that not through PIP, as on Windows.

Reply:
True.  I think that we can install pycryo via pip for all OS-es.

We should remove EXTRA_LIBRARIES as it introduced a long time ago when building with pip was complicated and should be kept only for the packages which don't work with pip.

-------

Also, I think that we should remove `PIP_LIBRARIES_OPENSSL_098` and just use the embedded openssl for those OSes.

There are so many exceptions and branches for how pachages are selected and I find it hard to read the code from here.

Reply:
Agreed, this is already spelled out in the ticket for a Python 3 package. But here I don't want to do too many things at the same time. For now, I have reversed your changes for Windows, Linux, and AIX 5.3.

Besides, I want to do those cleanups in dedicated PRs, so that they are easy to find in history. Maybe before forking for Python 3. Although I'm not so sure a forking is needed, we could just get rid of all the external sources and then advise using shallow clones.